### ISSUE TITLE
Does this angular library support server side rendering using angular universal?

### ISSUE THREAD

**Issue by r-kotagudem** [okta/okta-angular#101]
### Describe the feature request?

Does this angular library support server side rendering using angular universal?

### New or Affected Resource(s)

Does this angular library support server side rendering using angular universal?

**Comment by mraible** [comment]
No. Our Okta Auth JS library has server-side support. https://github.com/okta/okta-auth-js#node-js-and-react-native-usage
